file-formats/intm/type/zif_aff_intm_v1.intf.abap

```diff
@@ -0,0 +1,120 @@
+INTERFACE zif_aff_intm_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Intelligent Scenario Model</p>
+    "! This is an intelligent scenario model
+    BEGIN OF ty_intelligent_model,
+      "! <p class="shorttext">Model Name</p>
+      "! This is the model name
+      name                  TYPE c LENGTH 30,
+      "! <p class="shorttext">Parent</p>
+      "! This is the parent of the model
+      parent                TYPE c LENGTH 36,
+      "! <p class="shorttext">Model Type</p>
+      "! This is the model type
+      model_type            TYPE c LENGTH 256,
+      "! <p class="shorttext">Adapter ID</p>
+      "! This is the adapter ID
+      adapter_id            TYPE c LENGTH 32,
+      "! <p class="shorttext">Native Model Type</p>
+      "! This is the native model type
+      native_model_type     TYPE c LENGTH 256,
+      "! <p class="shorttext">Model Content</p>
+      "! This is the model content
+      model_specification   TYPE string,
+      "! <p class="shorttext">Model Definition</p>
+      "! This is the model definition
+      model_definition      TYPE string,
+      "! <p class="shorttext">State of model</p>
+      "! This is the state of the model
+      state                 TYPE c LENGTH 10,
+      "! <p class="shorttext">Signature</p>
+      "! This is the model signature
+      signature             TYPE string,
+      "! <p class="shorttext">Bindings</p>
+      "! This is the model bindings
+      bindings              TYPE string,
+      "! <p class="shorttext">Model flag</p>
+      "! This is the flag that indicate new model or enhancement(X)
+      new_model             TYPE abap_bool,
+      "! <p class="shorttext">ABAP Language Version</p>
+      "! This is the abap language version
+      abap_language_version TYPE c LENGTH 1,
```

Comment:
I guess this is the ABAP language version of the object itself . This information is stored in the `header`. Please don't add it here, again.

```suggestion
```